sh/scenarios/regression_4771.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/utils/main.sh
+source "$NCTL"/sh/views/utils.sh
+source "$NCTL"/sh/node/svc_"$NCTL_DAEMON_TYPE".sh
+source "$NCTL"/sh/scenarios/common/itst.sh
+
+#######################################
+# This test checks if the entry point of an installed contact can successfully be invoked.
+#
+# It covers against the regression captured in the #4771 ticket.
+#######################################
+function main() {
+    log "------------------------------------------------------------"
+    log "Regression 4771 test begins"
+    log "------------------------------------------------------------"
+
+    do_await_genesis_era_to_complete
+    TX_HASH=$(install_contract)
+    log "Install contract transaction hash: $TX_HASH"
+    await_n_blocks 2
+    ENTITY_CONTRACT=$(get_entity_contract $TX_HASH)
+    log "Contract entity: $ENTITY_CONTRACT"
+    INVOKE_ENTRY_POINT_RESULT=$(call_entry_point $ENTITY_CONTRACT)
+    log "Result of entry point invocation: $INVOKE_ENTRY_POINT_RESULT"
+
+    if [[ "$INVOKE_ENTRY_POINT_RESULT" == *"no such contract at hash"* ]]; then
```

Comment:
Not crucial for this particular issue, but I think it would be best here to check the return code of the client here as well so that we can catch other acceptor errors if they happen.

Reply:
Updated in 4420c798129db7188e99db3f483add1c4a256638.